life_expectancy/cleaning_strategies.py

```diff
@@ -0,0 +1,195 @@
+import argparse
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import Any
+
+import numpy as np
+import pandas as pd
+
+from life_expectancy.region import Region
+
+
+class AbstractCleaningStrategy(ABC):
+    """Abstract class that defines a strategy for cleaning some variants life_expectancy data."""
+
+    def __init__(self, col_dtypes=None):
+        super().__init__()
+        self.col_dtypes = {"year": np.int64, "value": float} if col_dtypes is None else col_dtypes
+
+    @abstractmethod
+    def load_data(self, data_path: Path, *args, **kwargs) -> pd.DataFrame:
```

Comment:
Why are the arguments " *args " and " **kwargs " required here? Or, are they just a good practice?